src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -61,16 +65,20 @@ public void storeMessage(Message message) {
             .addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
 
-    for (Entity entity : results.asIterable()) {
-      try {
+    for (Entity entity : results.asIterable())
+    {
+      try
+      {
         String idString = entity.getKey().getName();
         UUID id = UUID.fromString(idString);
+        String recipient = (String) entity.getProperty("recipient"); //I'll probably figure out what this does later
```

Comment:
Remove comment here. Remember that the comments you write will be shared with everyone. You are pushing to a team codebase. Comments that are for yourself are generally not allowed unless you need to write a to-do. It seems like this code gets the recipient from an entity in your data storage. But since we need the recipient as a string and getProperty doesn't return a string, we do a string cast here. This is similar to how we get text and timestamp below.